lib/main.coffee

```diff
@@ -47,6 +47,11 @@ module.exports =
     @subscribe atom.workspace.onDidChangeActivePane ->
       workspaceClassList.remove('vim-mode-plus-pane-maximized', 'hide-tab-bar')
 
+    @subscribe atom.workspace.onDidChangeActivePaneItem =>
+      if settings.get('automaticallyEscapeInesrtModeOnActivePaneItemChange')
```

Comment:
`automaticallyEscapeInesrtModeOnActivePaneItemChange`

Heads up: You spelled `Insert` as `Inesrt`. Just pointing it out now because it will be much easier to change before anyone starts using it.